src/Dns/Cname/Responses/UpdateResponse.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace ResellerClub\Dns\Cname\Responses;
+
+use ResellerClub\Response;
+
+class UpdateResponse extends Response
+{
```

Comment:
Does the API not have a response?

Reply:
Each request is either successful or not (exception thrown). No (useful) information is returned.

https://github.com/Pitchero/ResellerClub/blob/master/src/Orders/EmailForwarders/Responses/CreateResponse.php <-- just seems to duplicate the default action

(Let me know if I am missing something)

Reply:
On that example you've given there's still a response. Please note that the status in the abstract class might not be the same as the response status https://github.com/Pitchero/ResellerClub/blob/08a6ed13cb575b8ee1bb41fd380ebfde386907fe/src/Response.php#L45

Reply:
As discussed by phone, the base class handles this for us